.github/PROCESS.md

```diff
@@ -16,14 +16,35 @@ Keep guides short: we don't create rules; we automate.
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not end with a dot.
    It should not mention the issue number but describe the fixed issue instead.
-3. Pull request **description** typically just mentions the issue number like "Refs #123." or "Closes #456.".
+3. Pull request **description** must mention the issue number like "Refs #123." or "Closes #456.".
    Do not use `https://github.com/org/repo/issue/123` even if it is rendered the same on GitHub.
+   We provide a pull requests template that includes suggestions and readiness checklist.
    End sentences with a dot.
 4. Commit messages (both titles and bodies) are not important and can be "WIP" or anything else.
    All commits are always squashed on merge by GitHub, so there is no need to squash them manually, amend them, and/or do force pushes.
```

Comment:
That line is about the development process and it's not so important as it's automated by squash commit setting.